modules/gdscript/gdscript_function.cpp

```diff
@@ -1479,16 +1479,22 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 
 #ifdef DEBUG_ENABLED
 				GET_VARIANT_PTR(test, 1);
+				GET_VARIANT_PTR(message, 2);
 				bool result = test->booleanize();
 
 				if (!result) {
-
-					err_text = "Assertion failed.";
+					GET_VARIANT_PTR(message, 2);
```

Comment:
Either this one or the one above should be removed

Reply:
I don't understand - are you saying that the `message` variable should be removed?

Reply:
You use the `GET_VARIANT_PTR` macro twice with the same parameters. I don't think both calls are needed.

Reply:
Ah, I see what you mean. I should have looked at the full diff; not enough context here...